Content.Server/Chemistry/TileReactions/SpillTileReaction.cs

```diff
@@ -37,6 +37,11 @@ public FixedPoint2 TileReact(TileRef tile, ReagentPrototype reagent, FixedPoint2
                 var step = entityManager.EnsureComponent<StepTriggerComponent>(puddleUid);
                 entityManager.EntitySysManager.GetEntitySystem<StepTriggerSystem>().SetRequiredTriggerSpeed(puddleUid, _requiredSlipSpeed, step);
 
+                var slow = entityManager.EnsureComponent<SlowContactsComponent>(puddleUid);
+                var speedModifier = 1 - reagent.Viscosity;
```

Comment:
Slowoncontacts feels bad atm so without https://github.com/space-wizards/space-station-14/pull/16298 I'd be hesitant to merge this.

Reply:
merged